render_files.py

```diff
@@ -195,8 +212,14 @@ def main():
     # [TODO] - remove after fixing subsystem
     # create_services_config(work_dir, config_dir, openshift_release_image)
 
+    ignition_file = "%s/bootstrap.ign" % config_dir
+
     # update BMH configuration in boostrap ignition
-    update_bmh_files("%s/bootstrap.ign" % config_dir, cluster_id, inventory_endpoint, openshift_token(config_dir))
+    update_bmh_files(ignition_file, cluster_id, inventory_endpoint, openshift_token(config_dir))
+
+    if dhcp_allocation_file:
+        # Add dhcp allocation file if needed to ignition
+        add_dhcp_allocation_file(ignition_file, dhcp_allocation_file)
```

Comment:
Maybe you can also add a test ( the same way as for BMH), that will allow us testing and verifying ignition locally (on laptop)